### PR TITLE
Three Receivers Calibration: Add Reference to NanoVNA

### PR DESCRIPTION
**Pull request by biergaizi**
First, in the *Measuring a 4-Port With The 1.5-Port NanoVNA V2* guide, I added a suggestion for the user to read *Calibration With Three Receivers* first to familiarize themselves with the three-receiver VNA architecture. Also, I added a NanoVNA reference in the beginning of *Calibration With Three Receivers* as a new motivation for using the three-receiver calibration technique. This is another attempt to clarify the documentation per #629.

### CODE REVIEW

File: doc/source/examples/metrology/NanoVNA_V2_4port-splitter.ipynb [hunk 1]
```diff
@@ -13,7 +13,9 @@
    "id": "39fddba6",
    "metadata": {},
    "source": [
-    "The NanoVNA V2 is a 1.5-port VNA, meaning it can only measure the forward reflection and transmission coefficients $S_{11}$ and $S_{21}$ with its two ports. To measure the reverse coefficients $S_{12}$ and $S_{22}$ of a 2-port, it needs to be measured a second time with the port orientation physically flipped (or fake-flipped), as explained in [this example](TwoPortOnePath, EnhancedResponse, and FakeFlip.ipynb). The issue gets worse if the device under test (DUT) has even more ports, as described in [this example](./Measuring a Mutiport Device with a 2-Port Network Analyzer.ipynb).\n",
+    "The NanoVNA V2 is a 1.5-port VNA, meaning it can only measure the forward reflection and transmission coefficients $S_{11}$ and $S_{21}$ with its two ports. This architecture is known as a three-receiver VNA, still, fully-calibrated two-port measurements are possible despite this limitation. We recommended the readers to read [Calibration With Three Receivers](https://scikit-rf.readthedocs.io/en/latest/examples/metrology/Calibration%20With%20Three%20Receivers.html) first to familiarize themselves with the basic background.\n",
```
Comment (Ttl) [inline doc/source/examples/metrology/NanoVNA_V2_4port-splitter.ipynb:16]: I didn't realize that links could be given relative like below: `[this example](./Measuring a Mutiport Device with a 2-Port Network Analyzer.ipynb)`. Could you change the absolute links to skrf docs in this PR and your other recent PRs to relative ones? That way the links work also in local documentation.

**Review by Ttl — APPROVED** [review]

File: doc/source/examples/metrology/NanoVNA_V2_4port-splitter.ipynb (hunk 1 above)
Comment (biergaizi) [reply doc/source/examples/metrology/NanoVNA_V2_4port-splitter.ipynb:16]: I didn't realize it either, will do later. Sounds like a good idea to mass-convert the entire `doc/` directory.

Comment (biergaizi) [reply doc/source/examples/metrology/NanoVNA_V2_4port-splitter.ipynb]: Done, the links are all relative now.

**Review by jhillairet — APPROVED** [review]

**Review by Vinc0110 — APPROVED** [review]

**Comment by jhillairet** [comment]
@biergaizi ok for merging?

**Comment by biergaizi** [comment]
Yes.

**Pull request merged**